bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,202 @@ class CoffeeLintBear:
+                               apply_max_line_length_on_comments}}
+        if space_before_and_after_arrow:
+            coffee_configs["arrow_spacing"] = {"level": "ignore"}
+        if braces_spacing:
+            coffee_configs["braces_spacing"] = (
+                {"level": "ignore",
+                 "spaces": braces_spacing_width,
+                 "empty_object_spaces": spacing_in_empty_braces})
+        if camelcase_classes_name:
+            coffee_configs["camel_case_classes"] = {"level": "error"}
+        if spaces_before_and_after_colon:
+            coffee_configs["colon_assignment_spacing"] = (
+                {"level": "ignore",
+                 "spacing": {"left": spaces_before_colon,
+                             "right": spaces_after_colon}})
+        if end_file_with_a_single_newline:
+            coffee_configs["eol_last"] = {"level": "ignore"}
+        if newlines_after_classes:
+            coffee_configs["newlines_after_classes"] = (
+                {"value": number_of_lines_after_classes,
+                 "level": "ignore"})
+        if no_backticks:
+            coffee_configs["no_backticks"] = {"level": "error"}
+        if no_implicit_braces:
+            coffee_configs["no_implicit_braces"] = (
+                {"level": "ignore", "strict": True})
+        if no_function_call_without_parentheses:
+            coffee_configs["no_implicit_parens"] = (
+                {"strict": True, "level": "ignore"})
+        if no_interpolation_in_single_quotes:
+            coffee_configs["no_interpolation_in_single_quotes"] = (
+                {"level": "ignore"})
+        if no_stand_alone_at:
+            coffee_configs["no_stand_alone_at"] = {"level": "ignore"}
+        if no_tabs:
+            coffee_configs["no_tabs"] = {"level": "error"}
+        if no_throwing_strings:
+            coffee_configs["no_throwing_strings"] = {"level": "error"}
+        if no_trailing_semicolons:
+            coffee_configs["no_trailing_semicolons"] = {"level": "error"}
+        if no_trailing_whitespaces:
+            coffee_configs["no_trailing_whitespace"] = (
+                {"level": "error",
+                 "allowed_in_comments":
+                 apply_no_trailing_whitespaces_in_comments,
+                 "allowed_in_empty_lines":
+                 apply_no_trailing_whitespaces_in_empty_line})
+        if no_unnecessary_double_quotes:
+            coffee_configs["no_unnecessary_double_quotes"] = {"level": "ignore"}
+        if use_english_operator:
+            coffee_configs["prefer_english_operator"] = (
+                {"level": "ignore", "doubleNotLevel": "ignore"})
+        if space_around_operators:
+            coffee_configs["space_operators"] = {"level": "ignore"}
+        if space_after_comma:
+            coffee_configs["spacing_after_comma"] = {"level": "ignore"}
+
+        configs_json = json.dumps(coffee_configs)
+
+        return '--reporter=raw', '-f', configs_json, filename
```

Comment:
-f doesn want the json dump right? it wants the json file location AFAIK. shouldn't these configs go into `generate_config` then? see `RubocopBear` to see exactly how it works